### ISSUE TITLE
Status of Faster-RCNN using C# and GPU

### ISSUE THREAD

**Issue by mortengryning** [Microsoft/CNTK#2595]
Hi.

What is the current status for evaluating a Faster-RCNN model in C# using the GPU. Is it possible? Will it be possible? 

What about for C++?

I want to use the Faster RCNN implementation, but there is really no purpose if I cannot evaluate it on a GPU (compared to the normal RCNN implementation which I can evaluate on a very fast GPU server)

Br.
Morten Gryning

**Comment by Dozer3D** [comment]
I think the second part of this issue is the same as the feature request already logged in #2438.
Again, for the faster rcnn evaluation to be useful for most real world solutions, it needs to be GPU enabled. It would be good at least get a time estimate for this feature, so we can find alternative solutions if we can't get it soon.

**Comment by mortengryning** [comment]
Yes. If it's not implemented already, can you provide an estimate for when I will be implemented? Next release maybe?

**Comment by brantPTS** [comment]
Can you provide an update on the status of this feature? We could really use GPU enabled Faster-RCNN. I notice it was labeled feature request on April 10.

**Comment by augasur** [comment]
Any news on Faster-RCNN evaluation on GPU using CPP or C#?

**Comment by spandantiwari** [comment]
This is an open work item, but we don't have a timeline on this because of the other priorities that the team is working on currently. Apologies for the delay.
